### PR TITLE
Adds remove-range measure

### LINKED ISSUE #11: Function analysis - Remove Range
Analyze the python implementation of the string strip algorithm as can be found in https://github.com/SEFGroup23/algorithms-1/blob/master/algorithms/strings/strip_url_params.py

You identify a function/method with high complexity, and document its purpose, and why the complexity should be high (or not).
You also manually count the complexity of the function (on paper or as comments).

### PR DESCRIPTION
**Pull request by capitanu**
Fixes #11

### CODE REVIEW

**Review by medkamilmel — APPROVED** [review]
LGTM

File: manual_evaluations/part1_complexity/remove_range.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+List: [8, 13, 17, 4, 9, 12, 98, 41, 7, 23, 0, 92]
+remove_range(list, 3, 8);
+List becomes: [8, 13, 17, 23, 0, 92]
+
+legal range of the list (0 < start index < end index < size of list).
+"""
+
+# Questions in the lab pdf, part 1:
+#
+# 1. The results differ by 1 to the tool used. I had 1 higher CC than the tool.
```
Comment (ArnPellesGit) [inline manual_evaluations/part1_complexity/remove_range.py:19]: Yeah I don't understand how the tool I used ended up with 8. I get 9 too

**Review by ArnPellesGit — APPROVED** [review]
LGTM

File: manual_evaluations/part1_complexity/remove_range.py (hunk 1 above)
Comment (capitanu) [reply manual_evaluations/part1_complexity/remove_range.py:19]: I think it has something to do with for-loops.

**Pull request merged**